### ISSUE TITLE
Requirement file creation

### ISSUE THREAD

**Issue by sakshamsingh1** [micarraylib/micarraylib#6]
Easier to use the library if we create a requirements file of dependent libraries.

**Comment by iranroman** [comment]
I agree. This is a fundamental need of the project and we should address it soon. 

Current tested dependencies are `librosa >= 0.8.1`, `soundata >= 0.1.0`, `spaudiopy => 0.1.4` ( and `pytest => 6.2.5` for people who want to contribute). 

If you want you can include a `requirements.txt` as part of PR #5
